docs/whats-new.md

```diff
@@ -20,6 +20,12 @@ Release date: TBD, Target late Aug, 2018
         <p><i>Enhanced Multi-View API</i></p>
       </td>
     </tr>
+    <tr>
+      <td>
+        <img height=150 src="https://raw.github.com/uber-common/deck.gl-data/master/images/whats-new/contour-layer-cell-resize.gif" />
```

Comment:
Don't forget to resize the GIF to ensure reasonable loading speeds: https://github.com/uber-common/deck.gl-data